pkg/common/kafka/testing/mocks.go

```diff
@@ -18,47 +18,57 @@ package testing
 
 import (
 	"context"
-	"testing"
+	"fmt"
 
 	"github.com/Shopify/sarama"
+	"github.com/stretchr/testify/mock"
 )
 
 //
-// Mock ConsumerGroup Implementation
+// Mock Sarama ConsumerGroup Implementation
 //
 
-// Verify The Mock ConsumerGroup Implements The Interface
 var _ sarama.ConsumerGroup = &MockConsumerGroup{}
 
-// Define The Mock ConsumerGroup
 type MockConsumerGroup struct {
-	t           *testing.T
-	errorChan   chan error
+	mock.Mock
+	ErrorChan   chan error
 	consumeChan chan struct{}
 	Closed      bool
+	CloseErr    bool
 }
 
-// Mock ConsumerGroup Constructor
-func NewMockConsumerGroup(t *testing.T) *MockConsumerGroup {
-	return &MockConsumerGroup{
-		t:           t,
-		errorChan:   make(chan error),
+func NewStubbedMockConsumerGroup() *MockConsumerGroup {
+	mockGroup := &MockConsumerGroup{
```

Comment:
Why is this called `NewStubbedMockConsumerGroup()` instead of just `NewMockConsumerGroup()` ?

Reply:
Because originally there were two; will rename